war/.yarnrc.yml

```diff
@@ -1,8 +1,3 @@
-# Defines what linker should be used for installing Node packages (useful to
-# enable the node-modules plugin), one of: pnp, pnpm and node-modules.
-nodeLinker: node-modules
+nodeLinker: pnp
```

Comment:
IJ pesters me to run `yarn install` if I get a rid of node_modules but that does no longer work with pnp